### PR TITLE
QoL Fix for IC Issue Button

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
fix: The IC Issue prompt for admins has been improved, it now takes less clicks for users to respond to the message and in general has been souped up for a better experience around the board.
/:cl:

### CODE REVIEW

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -30,10 +30,10 @@
 
 		var/msg = "<font color='red' size='4'><b>- AdminHelp marked as IC issue! -</b></font><br>"
 		msg += "<font color='red'><b>Losing is part of the game!</b></font><br>"
-		msg += "<font color='red'>Your character will frequently die, sometimes without even a possibility of avoiding it. Events will often be out of your control. No matter how good or prepared you are, sometimes you just lose.</font>"
-
+		msg += "<font color='red'>Your character will frequently die, sometimes without even a possibility of avoiding it. Events will often be out of your control. No matter how good or prepared you are, sometimes you just lose.</font><br>"
+		msg += "<font color='red'>We have opened the admin complaints subforum for you to save you some time in creating your ban request.</font>"
 		C << msg
-
+		C << link("https://tgstation13.org/phpBB/viewforum.php?f=23")
```
Comment (coiax) [inline code/modules/admin/topic.dm]: > doesn't open up the admin complaint post for the admin in question
:-1:

Comment (Iamgoofball) [inline code/modules/admin/topic.dm]: Unfortunately we can't control the browser enough to properly fill out a template for them.

**Comment by Niknakflak** [comment]
Your memes are still not funny and you are still rude.

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (kevinz000) [inline code/modules/admin/topic.dm]: have it automatically select the first guy who namechecks if you're going to go all the way.

Comment (Iamgoofball) [inline code/modules/admin/topic.dm]: that's a good idea

**Comment by Iamgoofball** [comment]
I'm simply making sure Ban Requests 2.0 has a steady stream of traffic.

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/admin/topic.dm]: you could add a function to the forum to accept more parameters and then pass them trhough in the link.

you'd ave to ask mso to modify the forum code though lol

Comment (MrStonedOne) [inline code/modules/admin/topic.dm]: `https://tgstation13.org/phpBB/posting.php?mode=post&f=23`

**Comment by MrStonedOne** [comment]
goofball over reacting like always.


At least when I overreact people can still understand the general point i'm trying to make. I don't actually understand what point you are trying to make here.

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/admin/topic.dm]: @MrStonedOne I was referring more to adding parameters for topic body etc

**Comment by Iamgoofball** [comment]
@MrStonedOne 
pretending ban requests didnt just move operations to admin complaints is silly

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/admin/topic.dm]: yeah if I could fill out the template and stuff by default I'd do that

**Pull request closed without merging**